flink-core/src/main/java/org/apache/flink/api/common/io/FileInputFormat.java

```diff
@@ -55,20 +55,20 @@
  * {@link #nextRecord(Object)} and {@link #reachedEnd()} methods need to be implemented.
  * Additionally, one may override {@link #open(FileInputSplit)} and {@link #close()} to
  * change the life cycle behavior.
- * 
```

Comment:
Can you move the whitespace and checkstyle changes into a separate commit? This would make it much easier to review the PR. Thank you!